dashboard/test/helpers/curriculum_helper_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class CurriculumHelperTest < ActiveSupport::TestCase
+  setup do
+    @errors = Object.new
+    @errors.stubs(:add)
+    @object_to_validate = Object.new
+    @object_to_validate.extend(CurriculumHelper)
+    @object_to_validate.stubs(:errors).returns(@errors)
+  end
+
+  def refute_valid_key(key_value)
```

Comment:
beautiful, thanks for integrating the feedback!

Reply:
Thanks for taking a look!